build/KoreBuild.ps1

```diff
@@ -142,7 +122,5 @@ if (!(Test-Path $msbuildArtifactsDir))
 
 $msBuildArguments | Out-File -Encoding ASCII -FilePath $msBuildResponseFile
 
-# workaround https://github.com/dotnet/core-setup/issues/1664
```

Comment:
I tested. Its been fixed. https://github.com/dotnet/core-setup/pull/2055  It just looks like the bug hasn't been closed.